package Ques1;
import java.io.*;
class ExceptionHandling {
	public static void main (String[] args) {
	      int a=9,b=0;
	      try{
	          System.out.println(a/b);
	        }
	      
	      catch(ArithmeticException e){
	        
	      }
	      
	       finally {
	        	System.out.println("I will be there even if no one is there");
	        }
	    }
	}

